rest-api/test/unit_test/offline_test/biobank_samples_pipeline_test.py

```diff
@@ -52,6 +52,10 @@ def test_end_to_end(self):
       biobank_ids.append(participant.biobankId)
       self.assertEquals(summary_dao.get(participant.participantId).numBaselineSamplesArrived, 0)
     test1, test2, test3 = random.sample(_BASELINE_TESTS, 3)
+    test_list = [test1, test2, test3]
+    test1, test2, test3 = ['1PST8' if i=='1PS08' else '1SST8' if i == '1SS08' else i for i in \
```

Comment:
So this would have our tests only run with the old codes. I'm thinking we should either switch the tests to only use the new codes, or have a mix of both?